.github/workflows/security-code-scanner.yml

```diff
@@ -0,0 +1,33 @@
+name: 'MetaMask Security Code Scanner'
+
+on:
+  push:
+    branches: ['main']
+  pull_request:
+    branches: ['main']
```

Comment:
```suggestion
```

Is there a reason to not just run this on every PR?

Reply:
Not often, but we do sometimes have PRs that target other branches that then go into main later.

I just don't see a reason to limit where this is run!